keyboards/trainpad/info.json

```
@@ -0,0 +1,28 @@
+{
+    "keyboard_name": "TrainPad",
+	"manufacturer": "Ananya Kirti",
+	"url": "",
+	"maintainer": "Ananya Kirti",
+    "usb": {
+        "vid": "0x416B",
+        "pid": "0x0003",
+        "device_version": "0.0.1"
+    },
+	"processor": "atmega32u2",
```

Moving these out of `rules.mk` to here.

```suggestion
    "features": {
        "bootmagic": true,
        "mousekey": true,
        "extrakey": true,
        "console": false,
        "command": false,
        "nkro": false
    },
    "processor": "atmega32u2",
```